spring-cloud-azure-starters/spring-cloud-azure-starter-storage/README.md

```diff
@@ -0,0 +1,67 @@
+# Spring Cloud Azure Storage 
+
+The project implements Spring Resource abstraction for Azure Storage service which allows you to interact with [Azure Blob storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) using Spring programming model.
+
+## Samples 
+
+Please use this [sample](../../spring-cloud-azure-samples/spring-cloud-azure-storage-sample/) as a reference for how to use **Spring Cloud Azure Storage** in your projects. 
+
+## Feature List 
+
+- [Auto-configuration for Azure Blob storage](#auto-configuration-for-azure-blob-storage)
+- [Map with a resource](#map-with-a-resource)
+- [Read and write to a resource](#read-and-write-to-a-resource)
+- [Other operations](#other-operations) 
+
+### Auto-configuration for Azure Blob storage
+
+We provide a Spring Boot Starter [`spring-cloud-azure-starter-storage`](spring-cloud-azure-starters/spring-cloud-azure-starter-storage/) to auto-configure Azure Storge in your project. 
+
+If you are using Maven, add the following dependency to your project . 
+
+```
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-cloud-azure-starter-storage</artifactId>
+</dependency>
+```
+If you are using Gradle, add the following dependency to your project. 
+```
+dependencies {
+    compile group: 'com.microsoft.azure', name: 'spring-cloud-azure-starter-storage'
+}
+```
+
+The [`spring-cloud-azure-starter-storage`](spring-cloud-azure-starters/spring-cloud-azure-starter-storage/) provides the following configuration options in `application.properties`.
+
+Name | Description | Required | Default 
```

Comment:
Add property `spring.cloud.azure.storage.enabled`, which has default value `true` and is not required. This gives user the ability to disable auto config